### PR TITLE
Stage View for Images in Adaptive cards

### PR DESCRIPTION
**Pull request by pradeepananth**

### CODE REVIEW

File: msteams-platform/task-modules-and-cards/cards/cards-format.md [hunk 1]
```diff
@@ -213,7 +213,35 @@ A full width Adaptive Card appears as follows:
 If you have not set the `width` property to *Full*, then the default view of the Adaptive Card is as follows:
 ![Small width Adaptive Card view](../../assets/images/cards/small-width-adaptive-card.png)
 
+### Stage View for images in Adaptive cards
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/cards-format.md:216]: Please change to:

```suggestion
### Stage view for images in Adaptive Cards
```

File: msteams-platform/task-modules-and-cards/cards/cards-format.md [hunk 2]
```diff
@@ -213,7 +213,35 @@ A full width Adaptive Card appears as follows:
...
+You can also use the `msteams` property to selectively add the ability to enlarge images, in an Adaptive card, in stage view. Users would see an expand icon when they hover over the images, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/cards-format.md:217]:
```suggestion
In an Adaptive card, in stage view, you can use the `msteams` property to add the ability to enlarge images selectively. When users hover over the images, they would see an expand icon, for which the `allowExpand` attribute is set to `true`. For information on how to use the property, see the following example:
```

File: msteams-platform/task-modules-and-cards/cards/cards-format.md [hunk 3]
```diff
@@ -213,7 +213,35 @@ A full width Adaptive Card appears as follows:
...
+     ],
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.2"
+}
+```
+
+When users hover over the image, they would see an expand icon at top right corner of the image.
+![Adaptive card with expandable image](../../assets/images/cards/adaptivecard-hover-expand-icon.png)
+
+The image would be shown in stage view when clicking on the expand button.
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/cards-format.md:239]:
```suggestion
The image is shown in stage view when the user clicks on the expand button.
```

File: msteams-platform/task-modules-and-cards/cards/cards-format.md [hunk 4]
```diff
@@ -213,7 +213,35 @@ A full width Adaptive Card appears as follows:
...
+![Image expanded to stage view](../../assets/images/cards/adaptivecard-expand-image.png)
 
+In the stage view, users would be able to zoom in and zoom out of the image.
+You can choose which images in your Adaptive card need to have this capability.
+Please note that this ability applies only to image elements (Image type) in an Adaptive card.
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/cards-format.md:244]:
```suggestion
In the stage view, users can zoom in and zoom out of the image. You can select which images in your Adaptive card needs to have this capability.

> [!NOTE]
> Zoom in and zoom out capability applies only to the image elements (Image type) in an Adaptive card.
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Minor comments. Please verify.

File: msteams-platform/task-modules-and-cards/cards/cards-format.md (hunk 2 above)
Comment (pradeepananth) [reply msteams-platform/task-modules-and-cards/cards/cards-format.md]: this property is added  which enables users to see the image in stage view. Updated to reflect the same. please review and let me know if this doesnt look right.

**Review by surbhigupta12 — DISMISSED** [review]
Approving the PR.

**Review by ojasvichoudhary — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**